pkg/quarterdeck/auth.go

```diff
@@ -243,6 +280,9 @@ func (s *Server) Login(c *gin.Context) {
 	if out.AccessToken, out.RefreshToken, err = s.tokens.CreateTokenPair(claims); err != nil {
 		log.Error().Err(err).Msg("could not create jwt tokens on login")
 		c.JSON(http.StatusInternalServerError, api.ErrorResponse("could not create credentials"))
+
+		// increment failure count
+		metrics.FailedLogins.WithLabelValues("quarterdeck", "human", "jwt token error").Inc()
```

Comment:
Having these login failures is definitely going to help with having a good feeling of system health.